### ISSUE TITLE
GWS: Verify email can be exported from an account to another

### ISSUE THREAD

**Issue by 130s** [kinu-garage/hut_10sqft#1007]
Users at the school are interested in referring to the old emails that previous users of the same role received/sent. While my suggestion is to store future emals on external place such as `Google Groups`, the emails that were sent before G-Groups was enabled only exist on someones' `inbox`. We need a verify  a way to somehow move those emails out of inboxes. 

# CoS
- [ ] Multiple old emails on an account-A can be moved to somewhere else by batch.
- [ ] Ideally the user of account-A doesn't need to change any account settings, nor to be involved in operation.

**Comment by 130s** [comment]
TLDR; Data migration feature worked well. Explained well https://www.cubebackup.com/docs/tutorials/transfer-gsuite-data-to-another-account/ (I agree with this article that says the official document on Google.com is too verbose and hard to understand).

Some memo:
- Using "Data migration (Beta)" https://admin.google.com/u/5/ac/migrate/googleworkspace authorization request sent to the source (in my case account-a@xxxx.xxx), but the link to authorization requires admin console access, which the source account doesn't have so the web page fails to open. Trying to move forward, clicking "Authorize" fails
- Started conventional "data migration" https://admin.google.com/u/5/ac/dms and this seems to transfer all emails from account-a@xxxx.xxx to account-b@xxxx.xxx (same domain, under the same GWS org).
